README.md

```diff
@@ -1,37 +1,71 @@
-ember-test-waiters
-==============================================================================
+# ember-test-waiters
 
-[Short description of the addon.]
+This addon provides APIs to allow Ember testing to play nicely with other asynchronous
+events, such as an application that is waiting for a CSS3
+transition or an IndexDB transaction. Waiters runs periodically
+after each async helper (i.e. `click`, `andThen`, `visit`, etc) has executed,
+until a predetermined condition is met. After the waiters finish, the next async helper
```

Comment:
Hmm, this isn't _quite_ correct. Can we rephrase to make it clearer that the helpers themselves return a promise, and the "settledness" of these test waiters is part of the condition that determines when that promise resolves?

Reply:
Sure, but to be clear, this is taken verbatim from here:

https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/test/waiters.js#L8

Reply:
Updated. Let me know what you think.